Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11.11-slim
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py", "manage.py", "runserver", "0.0.0.0:8000"]

```

Comment:
The `CMD` instruction is attempting to run two scripts: `app/main.py` and `manage.py`. Docker's `CMD` can only execute one command. If you need to run multiple commands, consider using a shell script or `&&` to chain commands.